src/bin/pg_autoctl/monitor.c

```diff
@@ -2591,6 +2663,21 @@ monitor_stop_maintenance(Monitor *monitor, int nodeId)
 								   paramCount, paramTypes, paramValues,
 								   &context, &parseSingleValueResult))
 	{
+		if (monitor_retryable_error(context.sqlstate) &&
+			!pgsql_retry_policy_expired(retryPolicy))
+		{
+			int sleep = pgsql_compute_connection_retry_sleep_time(retryPolicy);
+
+			log_warn("Failed to start_maintenance of node %d on the monitor, "
+					 "retrying in %d ms.",
+					 nodeId, sleep);
+
+			/* we have milliseconds, pg_usleep() wants microseconds */
+			(void) pg_usleep(sleep * 1000);
+
+			return monitor_start_maintenance(monitor, nodeId, retryPolicy);
```

Comment:
```suggestion
			return monitor_stop_maintenance(monitor, nodeId, retryPolicy);
```

Reply:
Another good catch, thanks. Copy/paste and all.